cmd/eksctl-anywhere/cmd/upgradecluster.go

```diff
@@ -114,10 +133,32 @@ func (uc *upgradeClusterOptions) upgradeCluster(cmd *cobra.Command) error {
 		return err
 	}
 
+	_, err = time.ParseDuration(uc.cpWaitTimeout)
+	if err != nil {
+		logger.V(0).Info(fmt.Sprintf("Warning: Invalid %s value: %s, Using the default timeout instead: %s",
+			cpWaitTimeoutFlag, uc.cpWaitTimeout, clustermanager.DefaultControlPlaneWaitStr))
+		uc.cpWaitTimeout = clustermanager.DefaultControlPlaneWaitStr
+	}
+
+	_, err = time.ParseDuration(uc.externalEtcdWaitTimeout)
+	if err != nil {
+		logger.V(0).Info(fmt.Sprintf("Warning: Invalid %s value: %s, Using the default timeout instead: %s",
```

Comment:
Since this message `Warning: Invalid %s value: %s, Using the default timeout instead: %s` is repeated multiple times, can you define it as a const and just the const variable instead?

Something like

```go
# This can go at the top or in the constants file
const invalidTimeMessage = "Warning: Invalid %s value: %s, Using the default timeout instead: %s"

logger.V(0).Info(fmt.Sprintf(invalidTimeMessage, ...)
```